runtime/altair/src/lib.rs

```diff
@@ -1245,8 +1246,26 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
+	upgrade::Upgrade,
 >;
 
+/// Runtime upgrade logic
+mod upgrade {
+	use super::*;
+	use frame_support::{traits::OnRuntimeUpgrade, weights::Weight};
+
+	pub struct Upgrade;
+	impl OnRuntimeUpgrade for Upgrade {
+		fn on_runtime_upgrade() -> Weight {
```

Comment:
@mustermeiszer The migration is handled here in the runtime, since we need to coordinate some back-and-forth between the loans and interest-accrual pallets.